.github/workflows/manifest.yml

```diff
@@ -0,0 +1,18 @@
+name: Manifest
+on:
+  - pull_request_target
+
+jobs:
+  contribs:
+    runs-on: ubuntu-latest
+    name: Manifest
+    steps:
+      - name: Manifest
+        uses: zephyrproject-rtos/action-manifest@main
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          manifest-path: 'west.yml'
+          label-prefix: 'manifest-'
```

Comment:
This will add labels corresponding to each project that has its revision modified, for example:
`manifest-mbedtls`, `manifest-hal_nordic`